src/Components/__tests__/AboutDetails.spec.js

```diff
@@ -0,0 +1,53 @@
+import react from 'react';
+import { mount, shallow } from 'enzyme';
+
+import AboutDetails from '../AboutDetails';
+
+
+describe("<AboutDetails />", () => {
+    describe('ui', () => {
+        let wrapper;
+        beforeEach(() => {
+            wrapper = shallow(<AboutDetails />)
+        });
+
+        it('should render container', () => {
+            expect(wrapper.exists('.container-sm')).toEqual(true);
```

Comment:
let's add an `id` prop to the `AboutDetails` component and then in the test find the component by id.

Reply:
OK I wasn't sure how much you wanted me to change code in the Components themselves. I will use those going forward where it makes sense.